### PR TITLE
Enable EDGE functionality by default + Default COW for fragment cache

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: webapps/frontend/app/viewmodels/wizards/addvpool/index.js [hunk 1]
```diff
@@ -61,8 +61,8 @@ define([
         data.dtlEnabled(true);
         data.dtlMode({name: 'a_sync', disabled: false});
         data.dtlTransportMode({name: 'tcp'});
-        data.fragmentCacheOnRead(true);
-        data.fragmentCacheOnWrite(false);
```
Comment (kvanhijf) [inline webapps/frontend/app/viewmodels/wizards/addvpool/index.js:65]: data.js file still contains the reverse values, isn't this problematic?

File: ovs/lib/storagerouter.py [hunk 2]
```diff
@@ -924,6 +925,8 @@ def add_vpool(parameters):
         storagedriver_config.configure_event_publisher(events_amqp_routing_key=EtcdConfiguration.get('/ovs/framework/messagequeue|queues.storagedriver'),
                                                        events_amqp_uris=queue_urls)
         storagedriver_config.configure_threadpool_component(num_threads=16)
+        storagedriver_config.configure_network_interface(network_uri='tcp://{0}:{1}'.format(storagedriver.storage_ip,
```
Comment (kvanhijf) [inline ovs/lib/storagerouter.py:928]: Forgot to adapt some lines in this file
1331-1333

File: webapps/frontend/app/viewmodels/wizards/addvpool/index.js (hunk 1 above)
Comment (khenderick) [inline webapps/frontend/app/viewmodels/wizards/addvpool/index.js:65]: No, the `index.js`'s settings are used anyway, they immediately overwrite `data.js`'s settings. So in theory, most of `data.js`'s default could be removed at all.

File: ovs/lib/vdisk.py [hunk 3]
```diff
@@ -1173,7 +1173,7 @@ def dtl_checkup(vpool_guid=None, vdisk_guid=None, storagerouters_to_exclude=None
                         if dtl_host not in [sr.ip for sr in available_primary_storagerouters]:
                             VDiskController._logger.info('        Host not in available primary Storage Routers')
                             reconfigure_required = True
-                    if dtl_port != storage_drivers[0].ports[2]:
+                    if dtl_port != storage_drivers[0].ports['dtl']:
                         VDiskController._logger.info('        Configured port does not match expected port ({0} vs {1})'.format(dtl_port, storage_drivers[0].ports[2]))
```
Comment (kvanhijf) [inline ovs/lib/vdisk.py]: Logger line also contains a storagedriver port reference

File: ovs/lib/storagerouter.py (hunk 2 above)
Comment (khenderick) [inline ovs/lib/storagerouter.py:928]: Corrected, good catch.

**Pull request merged**